miner/currency_blocklist_test.go

```diff
@@ -70,6 +70,8 @@ func TestBlocklistAddAfterEviction(t *testing.T) {
 
 func TestBlocklistRemove(t *testing.T) {
 	bl := NewAddressBlocklist()
+
+	bl.Remove(feeCurrency1)
```

Comment:
Why is this needed?

Reply:
I wanted to check that there is no panic when attempting to remove from an empty blocklist. We have the `oldestHeader` field which is initially null, so just wanted to ensure that it was not accidentally accessed before being set.